.coafile

```diff
@@ -7,7 +7,7 @@ max_lines_per_file = 1000
 max_line_length = 120
 
 [all.python]
-bears = PycodestyleBear, PyDocStyleBear
+bears = PycodestyleBear
```

Comment:
Why remove this bear?

Reply:
I will add it back in.

Reply:
I've added it and it all passes coding style tests.